lambda/sendReminder/index.js

```
@@ -0,0 +1,203 @@
+            bulkJobSuccesses++;
+          }
+        } catch (err) {
+          jobError = 'The service was unsuccessful in leveraging GC Notify to send a bulk email.';
+          resData = String(err);
+          logger.error(jobError, err)
+          bulkJobFailures++;
+        }
+
+        try {
+          // Post a summary of job success/failure to db.
+          let jobObj = await postBulkReminderSummary(resData, jobError, chunk);
+          if (jobError) {
+            // Post alert to RocketChat channel if job fails.
+            request = {
+              postTitle: '@all **Day Use Pass - Bulk Email Service**',
+              postText: `A bulk email reminder job has failed: ${jobError} Details can be found under the job key listed below.`,
+              author_name: 'Day Use Pass',
+              author_icon: 'https://bcparks.ca/_shared/images/logos/logo-bcparks-v-200.png',
+              color: '#2D834F',
+              fields: [
+                {
+                  title: 'Job key:',
+                  value: `pk: ${jobObj?.pk}\nsk: ${jobObj?.sk}`,
+                  short: true
+                },
+                {
+                  title: 'Number of users affected',
+                  value: `${chunk.length - 1}`,
+                  short: true
+                }
+              ]
+            }
+            await rcPost(process.env.RC_ALERT_WEBHOOK_URL, process.env.RC_ALERT_WEBHOOK_TOKEN, request);
+          }
+        } catch (err) {
+          logger.error('Failed to document the bulk email job:', err);
+        }
+      }
+    } else {
+      logger.debug('No passes found for reminder service.')
+    }
+    const totalJobs = bulkJobSuccesses + bulkJobFailures;
+    logger.debug(`${totalJobs} job(s) run (${bulkJobSuccesses} succeeded, ${bulkJobFailures} failed).`)
+  } catch (err) {
+    // Something unknown went wrong.
+    return sendResponse(400, { msg: 'Something went wrong.', title: 'Operation Failed' });
```

We should emit error here to RC service, otherwise this would be silently failing without us knowing.